sentinel-demo/sentinel-demo-zookeeper-datasource/src/main/java/com/alibaba/csp/sentinel/demo/datasource/zookeeper/ZookeeperDataSourceDemo.java

```diff
@@ -1,57 +1,73 @@
+import com.alibaba.csp.sentinel.slots.block.degrade.DegradeRule;
 import com.alibaba.csp.sentinel.slots.block.flow.FlowRule;
-import com.alibaba.csp.sentinel.slots.block.flow.FlowRuleManager;
+import com.alibaba.csp.sentinel.slots.block.flow.param.ParamFlowRule;
+import com.alibaba.csp.sentinel.slots.system.SystemRule;
 import com.alibaba.fastjson.JSON;
 import com.alibaba.fastjson.TypeReference;
 
+import java.util.List;
+
 /**
  * Zookeeper ReadableDataSource Demo
  *
  * @author guonanjun
  */
 public class ZookeeperDataSourceDemo {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
         // 使用zookeeper的场景
         loadRules();
 
         // 方便扩展的场景
         //loadRules2();
     }
 
-    private static void loadRules() {
+    private static void loadRules() throws Exception {
 
         final String remoteAddress = "127.0.0.1:2181";
-        final String path = "/Sentinel-Demo/SYSTEM-CODE-DEMO-FLOW";
-
-        ReadableDataSource<String, List<FlowRule>> flowRuleDataSource = new ZookeeperDataSource<>(remoteAddress, path,
-                source -> JSON.parseObject(source, new TypeReference<List<FlowRule>>() {}));
-        FlowRuleManager.register2Property(flowRuleDataSource.getProperty());
-
-
+        final String pathFlow = "/Sentinel-Demo/SYSTEM-CODE-DEMO-FLOW";
+        final String pathSystem = "/Sentinel-Demo/SYSTEM-CODE-DEMO-SYSTEM";
+        final String pathDegrade = "/Sentinel-Demo/SYSTEM-CODE-DEMO-DEGRADE";
+        final String pathAuthority = "/Sentinel-Demo/SYSTEM-CODE-DEMO-AUTHORITY";
+        final String pathParam = "/Sentinel-Demo/SYSTEM-CODE-DEMO-PARAM";
+
+        new ZookeeperAllDataSource()
+                .serverAddr(remoteAddress)
+                .register(pathFlow, FlowRule.class, source -> JSON.parseObject(source, new TypeReference<List<FlowRule>>() {}))
+                .register(pathSystem, SystemRule.class, source -> JSON.parseObject(source, new TypeReference<List<SystemRule>>() {}))
+                .register(pathDegrade, DegradeRule.class, source -> JSON.parseObject(source, new TypeReference<List<DegradeRule>>() {}))
+                .register(pathAuthority, AuthorityRule.class, source -> JSON.parseObject(source, new TypeReference<List<AuthorityRule>>() {}))
+                .register(pathParam, ParamFlowRule.class, source -> JSON.parseObject(source, new TypeReference<List<ParamFlowRule>>() {}))
+                .start();
     }
 
-    private static void loadRules2() {
+    private static void loadRules2() throws Exception {
 
         final String remoteAddress = "127.0.0.1:2181";
         // 引入groupId和dataId的概念，是为了方便和Nacos进行切换
         final String groupId = "Sentinel-Demo";
-        final String flowDataId = "SYSTEM-CODE-DEMO-FLOW";
-        // final String degradeDataId = "SYSTEM-CODE-DEMO-DEGRADE";
-        // final String systemDataId = "SYSTEM-CODE-DEMO-SYSTEM";
+        final String flowDataIdFlow = "SYSTEM-CODE-DEMO-FLOW";
```

Comment:
The variable name `flowDataId` is better for code uniformity, I'm sorry for obsession...